src/zope/interface/tests/test_adapter.py

```diff
@@ -234,6 +234,17 @@ def test_unsubscribe_with_value_not_None_miss(self):
         registry.subscribe([IB1], None, orig)
         registry.unsubscribe([IB1], None, nomatch) #doesn't raise
         self.assertEqual(len(registry._subscribers), 2)
+    
+    def _instance_method_notify_target(self):
+        pass
```

Comment:
This is the line that is not being executed, which causes a test coverage regression.  You could drop a `# pragma: nocover` comment next to it, or use @jamadden's favorite trick of replacing the `pass` with a docstring, e.g. `"""example subscriber that is a bound method"""` or something.

Reply:
Or my other favorite trick, raising an exception if it's called. The best one here would be `self.fail("Example method, not intended to be called.")`

Reply:
[x] done